File: Pemrograman Berorientasi Objek/Materi/Pertemuan 5/multiple interfaces/php/Gudang.php
<?php

class Gudang extends Pabrik implements Aset{
	
	var $kodeAset;
	var $jenisAset;
	var $nilaiAset;
	var $kodeGudang;
	var $namaGudang;
	var $alamatGudang;
	
	function __construct(){
	}
	
	function setKodeAset($kodeAset){
		$this->kodeAset = $kodeAset;
	}
	
	function getKodeAset(){
		return $this->kodeAset;
	}
	
	function setJenisAset($jenisAset){
		$this->jenisAset = $jenisAset;
	}
	
	function getJenisAset(){
		return $this->jenisAset;
	}
	
	function setNilaiAset($nilaiGudang){
		$this->nilaiAset = $nilaiAset;
	}
	
	function getNilaiAset(){
		return $this->NilaiAset;
	}
	
	function setKodeGudang($kodeGudang){
		$this->kodeGudang = $kodeGudang;
	}
	
	function getKodeGudang(){
		return $this->kodeGudang;
	}
	
	function setNamaGudang($namaGudang){
		$this->namaGudang = $namaGudang;
	}
	
	function getNamaGudang(){
		return $this->namaGudang;
	}
	
	function setAlamatGudang($alamatGudang){
		$this->alamatGudang = $alamatGudang;
	}
	
	function getAlamatGudang(){
		return $this->alamatGudang;
	}
	
	function __destruct(){
	}
}

?>

File: Pemrograman Berorientasi Objek/Materi/Pertemuan 5/multiple interfaces/java/Gudang.java
class Gudang extends Pabrik implements Aset{
	
	private int kodeAset;
	private int jenisAset;
	private double nilaiAset;
	private int kodeGudang;
	private String namaGudang;
	private String alamatGudang;
	
	Gudang(){
	}
	
	public void setKodeAset(int kodeAset){
		this.kodeAset = kodeAset;
	}
	
	public int getKodeAset(){
		return this.kodeAset;
	}
	
	public void setJenisAset(int jenisAset){
		this.jenisAset = jenisAset;
	}
	
	public int getJenisAset(){
		return this.jenisAset;
	}
	
	public void setNilaiAset(double nilaiGudang){
		this.nilaiAset = nilaiAset;
	}
	
	public double getNilaiAset(){
		return this.nilaiAset;
	}
	
	public void setKodeGudang(int kodeGudang){
		this.kodeGudang = kodeGudang;
	}
	
	public int getKodeGudang(){
		return this.kodeGudang;
	}
	
	public void setNamaGudang(String namaGudang){
		this.namaGudang = namaGudang;
	}
	
	public String getNamaGudang(){
		return this.namaGudang;
	}
	
	public void setAlamatGudang(String alamatGudang){
		this.alamatGudang = alamatGudang;
	}
	
	public String getAlamatGudang(){
		return this.alamatGudang;
	}
	
}